Require Pillow-SIMD to prevent failing 'pip check' with included dependency Pillow<7.0
Author: Samuel Moors (Vrije Universiteit Brussel)
diff -ur detectron2-0.2.1.orig/setup.py detectron2-0.2.1/setup.py
--- detectron2-0.2.1.orig/setup.py	2020-07-30 09:23:23.000000000 +0200
+++ detectron2-0.2.1/setup.py	2020-09-05 13:50:26.421936000 +0200
@@ -175,7 +175,7 @@
     python_requires=">=3.6",
     install_requires=[
         "termcolor>=1.1",
-        "Pillow>=7.0",  # or use pillow-simd for better performance
+        "Pillow-SIMD>=7.0",  # or use pillow-simd for better performance
         "yacs>=0.1.6",
         "tabulate",
         "cloudpickle",
